pkgs/applications/backup/archivebox/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, python3Packages, }:
```

```suggestion
{ lib, fetchFromGitHub, python3 }:
```
facilitates overriding